server/plugins/client/bundle.js

```diff
@@ -65,9 +65,19 @@ function buildBundle (config, hoodieClientPath, callback) {
       return callback(error)
     }
 
-    var options = config.url ? '{url: "' + config.url + '"}' : ''
-    var initBuffer = Buffer('\n\nhoodie = new Hoodie(' + options + ')')
+    var initScript = '\n\n'
```

Comment:
Only to have some space between the client and initialization code. It also was the same way before :)